extensions/amp-position-observer/0.1/amp-position-observer.js

```diff
@@ -353,9 +353,9 @@ export class AmpVisibilityObserver extends AMP.BaseElement {
     let scene;
     if (this.targetId_) {
       scene = user().assertElement(
-        // TODO: In friendly iframe, getAmpDoc() incorrectly fetches the parent
-        // doc. Need to change it to return the FIE AMP doc.
-        this.getAmpDoc().getElementById(this.targetId_),
+        // TODO: Change back to this.getAmpDoc once ampdoc-fie experiment is
```

Comment:
Nit: Is there a tracking issue for this? If so please include ie. `// TODO(#####): `

Reply:
When cleaning up the experiment, a search of the experiment name `ampdoc-fie` should expose this.

Reply:
Please reference experiment cleanup issue in that case?